base/errorshow.jl

```diff
@@ -1086,6 +1092,28 @@ end
 
 Experimental.register_error_hint(methods_on_iterable, MethodError)
 
+# Display a hint in case the user tries to access non-member fields of container type datastructures
+function fielderror_hint_handler(io, exc)
+    @nospecialize
+    field = exc.field
+    type = exc.type
+    if type <: Dict
+        println(io,
+            """
+            \nDid you mean to access dict values using key: `$field` ?
+            Consider using `indexing` operation.
+            Example:
+                ```julia
```

Comment:
This renders as an indented fenced code block with fencing and without syntax highlighting. Perhaps print markdown or `printstyled`? Or don't use the code block.

Reply:
I remove code block for now. I tried experimenting with syntax highlighting but then I felt we should do it uniformly to other hints too. So I will reserve that to another PR may be if you find it reasonable.